packages/icon/README.md

```diff
@@ -0,0 +1,56 @@
+# Icon
+
+Mineral UI `Icon` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+This package provides a large number of prepackaged icon components as well as
+a generic Icon component to which you can provide your own SVG.
+
+
+## Installation
+
+1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
+
+2. Install the package
+
+  ```sh
+  npm install --save @mineral-ui/icon
+  ```
+
+
+## Usage
+
+### Prepackaged Icons
+
+```jsx
+import React from 'react';
+import IconHelp from '@mineral-ui/icon/components/IconHelp';
```

Comment:
the root of dist seems fine to me, but i haven't looked closely at the structure of the other components to know if it would be super inconsistent.

what paths are you concerned about potentially breaking?